examples/mc_sc_workflow_example.md

```diff
@@ -39,6 +39,8 @@ For any help, you could use the command `help`, for the exit just print `exit`
 
 **Step 3: Generate keypairs for the first Forger box in Sidechain**
```

Comment:
"Box" is still in the title of Step 3. It would be better to change to something like this "Generate keypairs for the first Forger account in Sidechain", so it is applicable to bith UTXO and Account models